sdk/communication/Azure.Communication.Identity/tests/CommunicationIdentityClient/CommunicationIdentityClientLiveTests.cs

```diff
@@ -477,6 +478,41 @@ public async Task GetTokenForTeamsUserWithWrongUserObjectIdShouldThrow()
             Assert.Fail("An exception should have been thrown.");
         }
 
+        [Test]
+        [TestCase(ServiceVersion.V2021_03_07, TestName = "CreateIdentityWithServiceVersion_V2021_03_07")]
+        [TestCase(ServiceVersion.V2022_06_01, TestName = "CreateIdentityWithServiceVersion_V2022_06_01")]
+        [TestCase(ServiceVersion.V2022_10_01, TestName = "CreateIdentityWithServiceVersion_V2022_10_01")]
+        public async Task CreateIdentityWithDifferentServiceVersions(ServiceVersion version)
+        {
+            try
+            {
+                CommunicationIdentityClient client = CreateClient(default, version);
+                CommunicationUserIdentifier userResponse = await client.CreateUserAsync();
+                Assert.IsNotNull(userResponse);
+            }
+            catch (Exception ex)
+            {
+                Assert.Fail($"Unexpected error: {ex}");
+            }
+        }
+
+        [Test]
+        public void CreateClientWithIncorrectServiceVersionShouldThrow()
+        {
+            try
+            {
+                ServiceVersion invalidVersion = (ServiceVersion)4;
```

Comment:
I'd use something like 120 or -1 that'd most likely never be valid..to prevent throwing once we add another valid version.

Reply:
Done